common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/MojoScorer.java

```diff
@@ -136,19 +140,20 @@ public ContributionResponse computeContribution(ContributionRequest request) {
     ShapleyType requestedShapleyType = shapleyType(request.getRequestShapleyValueType());
     switch (requestedShapleyType) {
       case TRANSFORMED:
-        MojoFrame requestFrame = contributionRequestConverter
-                .apply(request, pipelineShapley.getInputFrameBuilder());
-        return contribution(requestFrame);
+        MojoFrame requestFrame1 = contributionRequestConverter
```

Comment:
nit: how is `requestFrame1` different from `requestFrame2`? seems like the it is a result which is returned without being carried past the switch. I think probably it is better to have just `requestFrame`